receiver/filelogreceiver/README.md

```diff
@@ -98,6 +98,15 @@ The header lines are not emitted by the receiver.
 
 Many parsers operators can be configured to embed certain followup operations such as timestamp and severity parsing. For more information, see [complex parsers](../../pkg/stanza/docs/types/parsers.md#complex-parsers).
 
+### Log Rotation
+
+Log Rotation is supported for File Log Receiver, but the following settings need to be configured:
+
+- `poll_interval`: `10ms` (can adjust to fit your use case)
+- `start_at`: `beginning`
+
+External software needs to be used in order to do log rotation. For example, [logrotate](https://linux.die.net/man/8/logrotate) can be used to handle log rotation.
```

Comment:
Since the file log receiver doesn't have any built-in log rotation, I wanted to state that external software needs to be used to perform log rotation while also giving them an example of software they can use to do it. But if you think it's unnecessary, I can remove that statement.

Reply:
I think it's fine to specifically call out that we can read files that are being rotated but I don't think it's necessary to point out that the receiver doesn't actually write/copy/move files.